.vscode/tasks.json

```diff
@@ -0,0 +1,19 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
```

Comment:
This is nice!

Maybe we can see if there is any task we can have to compile the modified files before run the tests.